bin/check-and-renew

```diff
@@ -93,9 +93,9 @@ function check_running_job {
 
       # Check the age of the last log
       # If there are only two lines, there are no logs
-      if [[ $ok_to_restart == "YES" && $((`log_count` > 2)) == '1' ]]; then
+      if [[ $ok_to_restart == "YES" && $(($(log_count) > 2)) == '1' ]]; then
         # if the timestamp is younger than 15 mins from start of script
```

Comment:
```suggestion
        # if the timestamp is younger than 40 mins from start of script
```

Reply:
Oh... is this a long running change?

Reply:
some data.json take longer to download.
https://gsa-tts.slack.com/archives/C2N85536E/p1692638742516909